spec/unit/properties_spec.rb

```diff
@@ -17,6 +18,15 @@
 
     it { is_expected.to track_modifications }
     it { is_expected.to track_creations }
+
+    describe 'negative case' do
+      subject { expect(Post.new).to define_property(:description, :Integer) }
+
+      it 'should fail on invalid property type' do
+        expect { subject }.to raise_error RSpec::Expectations::ExpectationNotMetError,
+                                          'expected the Post model to have a `Integer` property description'
```

Comment:
@cheerfulstoic the reason that we did it that way is because the intended audience is those that are testing their own code. Part of that is when things go awry, you get a good failure message. By simply writing specs in the exact same fashion as users would, I think we're losing the explicitness of what happens when things fail (and what the library tells users)